src/constants.h

```diff
@@ -71,8 +71,6 @@ CONSTEXPR(uint32_t, surveyTimerMsec, 300000, 4000, 0)
   inline type functionName() { return inProduction() ? prod : beta; }
 
 constexpr const char* API_PRODUCTION_URL = "https://vpn.mozilla.org";
-constexpr const char* API_STAGING_URL =
-    "https://stage-vpn.guardian.nonprod.cloudops.mozgcp.net";
```

Comment:
We still use this in two places. We should remove it there too.
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/platforms/wasm/wasmnetworkrequest.cpp#L59
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/tests/unit/mocnetworkrequest.cpp#L40